traceapp/tmpl.go

```diff
@@ -97,7 +98,12 @@ func (a *App) parseHTMLTemplates(sets [][]string) error {
 			"dict":              dict,
 		})
 		for _, tmp := range set {
-			tmplBytes, err := tmpl.Asset(tmp)
+			tmplFile, err := tmpl.Assets.Open("/" + tmp)
+			if err != nil {
+				return fmt.Errorf("template %v: %s", set, err)
+			}
+			tmplBytes, err := ioutil.ReadAll(tmplFile)
+			tmplFile.Close()
```

Comment:
Yeah, I think that would make sense if we did it in more than a few places -- but IMO it's a bit easier to read these standard lines rather than not strictly knowing what `vfsutil.ReadFile` (despite obvious name) does.